analysis/pf_variables_library.py

```diff
@@ -24,19 +22,24 @@ def count_past_events(index_date, selected_events, codelist, num_months):
         .count_for_patient()
     )
 
+# Function to get events linked to a specified codelist
+def get_events(event_frame, codelist):
+    selected_events = event_frame.where(
+        event_frame.snomedct_code.is_in(codelist)
+    )
 
-# Function to get consultation IDs linked to a specified codelist
-def get_consultation_ids(clinical_events, codelist):
-    consultation_ids = clinical_events.where(
-        clinical_events.snomedct_code.is_in(codelist)
-    ).consultation_id
-
-    return consultation_ids
-
+    return selected_events
 
 # Function to get events with specific consultation IDs
 def get_consultationid_events(event_frame, consultation_ids):
```

Comment:
just an attempt to use a similar naming approach for our two functions that we use to select events 

```suggestion
def select_events_by_consultation_id(event_frame, consultation_ids):
```